### PR TITLE
closes #91 | Suggested Tailors

### LINKED ISSUE #91: If a tailor should reject an order,  the customer should be presented with three suggestion of tailors near him/her

### PR DESCRIPTION
**Pull request by Saint-lawal**
*Description of Task*
- Get next suggested tailors

### CODE REVIEW

File: src/controllers/Tailors.js [hunk 1]
```diff
@@ -166,6 +166,55 @@ class Tailors {
   }
 
   /**
+   * Handles retrieving tailors based on first tailor area
+   * @param {Object} req 
+   * @param {Object} res 
+   */
+  static retrieveOtherTailors(req, res) {
```
Comment (adetola-ralph) [inline src/controllers/Tailors.js]: check that the tailors don't exist in the order `rejectedTailorId` property. I guess the would be done after my PR #113 is merged in

Comment (Saint-lawal) [reply src/controllers/Tailors.js]: Hehn? explain what you mean @adetola-ralph

Comment (bolorundurowb) [reply src/controllers/Tailors.js]: @Saint-lawal his PR has been merged in. @adetola-ralph oya explain

Comment (adetola-ralph) [reply src/controllers/Tailors.js]: We need to take into account the tailors in order's rejectedTailorIds list so we don't send a tailor that has already rejected an order

Comment (adetola-ralph) [reply src/controllers/Tailors.js]: I think I will add it in

Comment (Saint-lawal) [reply src/controllers/Tailors.js]: Wait, what's this part about? Cause if you check the controller, you would see that a list of rejected tailors Id's is being sent as well @bolorundurowb @adetola-ralph

**Comment by bolorundurowb** [comment]
@Saint-lawal please update this branch so I can review and merge

**Pull request closed without merging**